sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko

```diff
@@ -0,0 +1,49 @@
+import { getAsObject, getAsArray, get } from '@base-cms/object-path';
```

Comment:
`getAsArray` and `get` are not used. Remove them from the import.

Reply:
Fixed